templates/zerver/register.html

```diff
@@ -100,7 +100,7 @@
             <label for="id_team_name" class="inline-block label-title">{{ _('Organization name') }}</label>
 
             <div class="help-box margin-top">
-                {{ _('This can be changed later on the settings page.') }}
+                {{ _('This can always be changed later. Keep it short!') }}
```

Comment:
I feel it's kind of weird that we have an organization name field which encourage short names as well as another field called short name. (This field would be called Subdomain instead of Short name when subdomains are enabled). Is this change really necessary?